lib/timescaledb/rails/models/chunk.rb

```diff
@@ -3,7 +3,7 @@
 module Timescaledb
   module Rails
     # :nodoc:
-    class Chunk < ::ActiveRecord::Base
+    class Chunk < Railtie.config.record_base.constantize
```

Comment:
need to allow the user to specify an abstract superclass, since that's the level that connections are configured

Reply:
Testing new changes I merged today into main, using `bin/console` generates this error

```
uninitialized constant Timescaledb::Rails::Railtie (NameError)
```

Reply:
That makes sense, can do that! Will rebase main & see what's changed there